kedro/framework/cli/starters.py

```diff
@@ -79,9 +171,7 @@ def create_cli():  # pragma: no cover
 @click.option("--starter", "-s", "starter_name", help=STARTER_ARG_HELP)
 @click.option("--checkout", help=CHECKOUT_ARG_HELP)
 @click.option("--directory", help=DIRECTORY_ARG_HELP)
-def new(
-    config_path, starter_name, checkout, directory, **kwargs
-):  # pylint: disable=unused-argument
+def new(config_path, starter_name, checkout, directory, **kwargs):
```

Comment:
```suggestion
def new(config_path, starter_alias, checkout, directory, **kwargs):
```
Needs to be changed in a few places.